source/development/client-api/cpp-client-programmers-guide.rst

```diff
@@ -1058,6 +1058,42 @@ alive.
         std::cout << "at least one dead/busy/locked/... device" << std::endl;
     }
 
+Enabling and disabling group members
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Devices belonging to a group can be temporarily excluded from all operations
+performed on the group using the *Group::disable* and *Group::enable* calls.
+
+Device name passed to the *disable* (*enable*) methods can contain
+wildcards (``*``).
+Note that only the first matching device will be disabled (enabled).
+
+The search algorithm is breadth-first search. All group elements are searched
+(in the insertion order) for a match before descending recursively to
+sub-groups. Recursive search can be disabled with a flag.
+
+Below is an example using the gauges group:
+
+.. code-block:: cpp
+    :linenos:
+
+    // will disable: inst-c01/vac-gauge/penning-01
+    gauges->disable("inst-c01/*/penn");
+
+    // will disable nothing
+    const bool forwarded = true;
+    gauges->disable("inst-c01/vac-gauge/pirani-01", not forwarded);
+
+    // will disable: inst-c01/vac-gauge/pirani-01
+    gauge_family->disable("inst-c01/vac-gauge/pirani-01");
+
+    // will enable: inst-c01/vac-gauge/penning-01
+    gauges->enable("inst-c01/*");
+
+    // entries for disabled elements will be included in the result set,
+    // however they will not have a value and will be marked as disabled
```

Comment:
How are they marked as disabled? Exception? A flag?
Can we add an example here showing how the user can ignore the attributes which are disabled in the result set?

Reply:
I've added `group_element_enabled()` check to the example. Is it ok now?